prov/mrail/src/mrail.h

```diff
@@ -128,16 +128,41 @@ mrail_match_recv_handle_unexp(struct mrail_recv_queue *recv_queue, uint64_t tag,
 			      uint64_t addr, char *data, size_t len, void *context);
 
 /* mrail protocol */
-#define MRAIL_HDR_VERSION 1
+#define MRAIL_HDR_VERSION 2
+
+#define MRAIL_PROTO_EAGER	0
+#define MRAIL_PROTO_RNDV	1
+
+#define MRAIL_RNDV_REQ		0
+#define MRAIL_RNDV_ACK		1
```

Comment:
nit: I think these could be defined using enums

Reply:
That can be done.